Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -9828,6 +9828,37 @@ script:
       type: Number
       required: true
       isArray: false
+  - name: pan-os-get-certificate-info
+    description: 'Gathers name, expiration date and expiration status of certificates configured locally on a Firewall or pushed from Panorama, as seen under Certificate Management.'
```

Comment:
Gathers the name, expiration date, and expiration status of certificates configured locally on a Firewall or pushed from Panorama, as seen under Certificate Management.'